script/run-cypress-tests-circle.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Use the junit reporter & save results in './test-results'.
+reporterArgs="--reporter junit --reporter-options \"mochaFile=test-results/e2e-test-output-[hash].xml\""
+    
+# Use the CircleCI CLI to get Cypress tests & split them on different machines by timing. Commas are inserted as the delimiter for file paths.
```

Comment:
What does "split by timing" mean?

Reply:
"Split by timing" separates tests on multiple machines based on the previous run's test data. So CircleCI looks at the timing data of the previous run, and uses this to determine how to run tests on different machines in the most time efficient way. Here is some [info](https://circleci.com/docs/2.0/parallelism-faster-jobs/#splitting-by-timing-data).